packages/suite/src/utils/wallet/solanaUtils.ts

```diff
@@ -216,9 +233,9 @@ export const buildTokenTransferTransaction = async (
     let finalReceiverAddress = toAddress;
     if (isReceiverAddressSystemAccount) {
         // Step 3: If not, check if the receiver owns an associated token account
-        if (toTokenAccounts && toTokenAccounts.length > 0) {
+        if (toTokenAccount != null) {
```

Comment:
We're not using loose equality operators, also there should never be `null` in this case.
```suggestion
        if (toTokenAccount) {
```

Reply:
Updated in https://github.com/trezor/trezor-suite/pull/10164/commits/921b7204a47adeb817fba4d818ca5ce87e2f1ba0#diff-143383856d0b16d79c6ef3b53799bfd67f1b72d5558060a3ca90fdc262ad5982R230. (again, sorry for the force push).